pulp_python/app/pypi/views.py

```diff
@@ -189,13 +212,28 @@ def create_group_upload_task(self, cur_session, repository, artifact, filename,
 class SimpleView(PackageUploadMixin, ViewSet):
     """View for the PyPI simple API."""
 
-    permission_classes = [IsAuthenticatedOrReadOnly]
```

Comment:
We don't need to specify the APfromDB because that's the globally configured default, right?

Reply:
Correct.